sledge/src/fol/context.ml

```diff
@@ -9,7 +9,8 @@
 
 open Exp
 
-(* Solution Substitutions =================================================*)
+(* Solution Substitutions
+   =================================================*)
```

Comment:
As a reminder for the Infer team, these comments may need to be re-styled in the future.